src/ProgressOnderwijsUtils/Win32/VirusScan.cs

```diff
@@ -0,0 +1,55 @@
+﻿#nullable disable annotations
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.InteropServices;
+using Microsoft.Windows.Sdk;
+using ProgressOnderwijsUtils;
+
+namespace ProgressOnderwijsUtils.Win32
+{
+    public static class VirusScan
+    {
+        public static bool IsMalware(string content, string contentName)
+        {
+            using var scanSession = new VirusScanSessie("Progress.Net");
+            PInvoke.AmsiScanString(scanSession.Context, content, contentName, IntPtr.Zero, out var result);
+            return ResultIsMalware(result);
+        }
+
+        public static bool IsMalware(byte[] buffer, string contentName) => IsMalwareUnsafe(buffer, contentName);

```

Comment:
Deze wrapper is zinloos; unsafe slaat op de method inhoud, niet wie het aan mag roepen.